### ISSUE TITLE
SonoHana 9

### ISSUE THREAD

**Issue by jurassicplayer** [BASLQC/vnds#2]
tl;dr: Here's a SonoHana 9 download link, add it as you see fit.
http://www.mediafire.com/file/nzga2n9uag16328/SonoHana9.zip/file

Storytime:
So, recently someone contacted me asking to talk about VNDS stuff (me being the derp who wrote the crummy tutorial, CXBlackCatXD). He was trying to figure out a simple issue that we ironed out over a number of emails, and he then also asked about SonoHana 9, which to my knowledge was never posted anywhere (chapters 1-8 were, though they don't seem to be in that nice google drive you have here either). I told him I'd see if the guy who originally ported them over was alive (Raidosu), which he was, and lo-n-behold he just happened to have SonoHana 9 already converted as well.

He gave no guarantees that it's complete/playable/etc. and I won't either, but in theory it works. He uploaded it for me to pass around and since this project seems to be trying to keep this stuff alive, I'd thought I'd spread it here. 

Side note: 
Thanks to whoever put up that backup for the vnds interpreter apk, I always wanted the latest version of that.

**Comment by antonizoon** [comment]
Good to hear, thank you. You know people can always make new vnds scripts. Though visual novels are getting more advanced with branching and they are typically encoded in some weird form.

**Comment by poudink** [comment]
yep it's glitchy alright. you can't get past the title screen because of a script error in main.scr. seems to be working fine once you correct that though.
